qp/pdf.py

```diff
@@ -222,9 +231,10 @@ def interpolate(self, using=None, vb=True):
             if self.quantiles is None:
                 self.quantiles = self.quantize()
 
-            self.difs = self.quantiles[1:]-self.quantiles[:-1]
-            self.mids = (self.quantiles[1:]+self.quantiles[:-1])/2.
-            self.vals = (1.0/(len(self.quantiles)+1))/self.difs
+            #self.difs = self.quantiles[0][1:]-self.quantiles[0][:-1]
+            #self.mids = (self.quantiles[0][1:]+self.quantiles[0][:-1])/2.
+            self.mids = self.quantiles[0]
```

Comment:
Much neater! Feel free to remove the commented out parts, they're in the history.